### ISSUE TITLE
s-tar causes evaluation errors

### ISSUE THREAD

**Issue by peti** [NixOS/nixpkgs#41479]
`s-tar` from https://github.com/NixOS/nixpkgs/pull/40771 causes evaluation errors in `master` @ 1d1a4bfb392c7d5d2ca5e0ccd66624b7c9f3aa7e (and before):

~~~
$ nix-env -qaP --xml --meta 2>&1 >/dev/null
derivation 's-tar-1.5.3' has invalid meta attribute 'override'
derivation 's-tar-1.5.3' has invalid meta attribute 'overrideDerivation'
~~~

**Comment by peti** [comment]
ping @xeji

**Comment by xeji** [comment]
Interesting. There was no eval error on ofborg when #40771 was merged, and all hydra builds succeded until at least master 84440379bade6fc7c42b11377e0595e6c3ff7117 (6 hours ago). I'll look into it.

**Comment by xeji** [comment]
Turns out it used `callPackages` instead of `callPackage`. Don't know why that produces this error.
Fixed in f44c16401e71aac01028a75f46bd52480f9b49b0
